x11-wm/cwm/cwm-6.6.ebuild

```diff
@@ -1,35 +1,34 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 2020 Gentoo Authors
```

Comment:
you should leave original year intact.

Reply:
I've updated the years to the correct range, hopefully it is OK now.